core/templates/dev/head/pages/exploration_player/ExplorationPlayerStateService.ts

```diff
@@ -17,6 +17,14 @@
  *  like engine service.
  */
 
+
+oppia.constant('EXPLORATION_MODE', {
+  EXPLORATION: 'exploration',
+  PRETEST: 'pretest',
+  QUESTION_PLAYER: 'question_player',
+  OTHER: 'other'
```

Comment:
What's `other` here?

Reply:
Other is a catch-all for any mode that cannot be explained in one of the above options.